### ISSUE TITLE
A name that doesn't parse correctly

### ISSUE THREAD

**Issue by sandbergja** [inukshuk/bibtex-ruby#149]
Thank you for this wonderful gem!  I am working with a specific author's name: `Rongiéras d'Usseau, Emmanuel`

When I try to run `BibTeX::Name.parse 'Rongiéras d\'Usseau, Emmanuel'`, I get `nil`.  It appears that this specific combination is not able to be parsed correctly:

- Name in indirect order (surname first)
- Two surnames
- The first surname begins with a capital letter
- The second surname begins with a lower case letter

Would you be interested in a pull request on this issue?  If so, would I modify `names.y` and provide a test?

**Comment by inukshuk** [comment]
Good catch! Yes, the name parser's grammer is in the `names.y` file. I suspect the issue is the lower case letter at the start of the second surname -- this might be tricky (for reference you could take a look at the namae gem, which evolved out of this name parser, but I doubt that we covered this case there).
